Tools/Scripts/cross-toolchain-helper

```diff
@@ -67,6 +67,11 @@ def configure_logging(selected_log_level='info'):
     logger.setLevel(log_level)
     return handler
 
+def set_env_var(env_key, env_value, loginfo=False):
+    os.environ[env_key] = env_value
+    if loginfo:
+        _log.info('export {env_key}="{env_value}"'.format(env_key=env_key, env_value=env_value))
```

Comment:
Side question, why using format strings in this stuff? Why not f-strings? Is this supposed to run with python2 which is EOL since Jan 1st 2020?

Reply:
Not good reason to do that, just a bad habit.
I would try to change the code to f-strings on an upcoming patch soon.